configuration.go

```diff
@@ -270,6 +272,12 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 
+	//default Swarm
+	var defaultSwarm provider.Swarm
+	defaultSwarm.Watch = true
+	defaultSwarm.ExposedByDefault = true
+	defaultSwarm.Endpoint = "unix:///var/run/docker.sock"
```

Comment:
@diegofernandes What if we want to run traefik as Swarm service. I don't think we can bind the unix socket into a service container. Is there a way to achieve that? Thanks!